platforms/scripts/afu_platform_config

```diff
@@ -804,19 +806,19 @@ def findAfuIfcs(db_path):
     afus = set()
     # Walk all the directories
     for db_dir in db_path:
-        try:
-            # Look for JSON files in each directory
-            for fn in os.listdir(db_dir):
-                if fn.endswith('.json'):
+        # Look for JSON files in each directory
+        for fn in os.listdir(db_dir):
+            if fn.endswith('.json'):
+                try:
```

Comment:
Same thing here, use glob?
Also, would it make sense to refactor the process of finding json files in a directory tree?